### ISSUE TITLE
Rework of selectionBackground & addition of selectionForeground

### ISSUE THREAD

**Issue by linuxluzer** [microsoft/terminal#4971]
![selectionBackground-selectionForeground](https://user-images.githubusercontent.com/59659765/76827230-226a4c80-67fd-11ea-882b-dedfd0fd0f86.PNG)

While making the new WT look and feel like Powershell, the `selectionBackground ` property seems affected by an acrylic effect that reduces contrast for accessibility and causes an ugly sensation whit some color schemes. It should be a `true`/`false` option that triggers the cmd/powershell mode (full opacity). In that case, a `selectionForeground ` could be added too, in order to address the parity features with conhost.exe.

A technical implementation in `profiles.json` could be something like this (don´t have enough C++ knowledge for real code):

```
"profiles":
    {
        "list":
        [
            {
                "name": "Windows PowerShell",
                "commandline": "powershell.exe",
                "selectionOpacity": true,
                "selectionBackground": "#fedba9",
                "selectionForeground": "#ffffff",
                "hidden": false
             }
        ]
    }
```

**Comment by DHowett-MSFT** [comment]
Thanks for the request! This is, at its heart, a /dupe of #3580 and #3561. :smile:
